taxi/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+
```

Comment:
There is no need to add blank line between imports that belong to the same import group, remove it